e2e/Dockerfile

```diff
@@ -4,4 +4,6 @@ RUN apk --no-cache add git
 
 WORKDIR /e2e
 
+RUN GO111MODULE=on go get -u github.com/google/go-containerregistry/cmd/crane
```

Comment:
This should be pinnned to a specific version I think.
Two options:
```
Create a go.mod you use to track your globally installed tools, such as in ~/global-tools/go.mod, and cd to that directory prior to running go get or go install for any globally installed tools.

Create a go.mod for each tool in separate directories, such as ~/tools/gorename/go.mod and ~/tools/goimports/go.mod, and cd to that appropriate directory prior to running go get or go install for the tool.
```

Reply:
Can't we control the versions using the @version at the end of `go get` command and manage them only inside the Dockerfile? Since we're only using crane's binary, I'm not sure if go.mod is needed.

Reply:
Good idea. Simpler and same effect. +1

Reply:
Since crane doesn't have specific versions, I used `34fb8ff` which is the hash for insecure flag PR's merge commit.